IBPSA/Electrical/Data/PV/SingleDiodeData.mo

```diff
@@ -0,0 +1,58 @@
+within IBPSA.Electrical.Data.PV;
+record SingleDiodeData
+  "Basic record of a PV module with single diode base model"
+  extends Modelica.Icons.Record;
+  extends IBPSA.Electrical.Data.PV.Generic;
+
+  parameter Modelica.Units.SI.Efficiency eta0
+    "Efficiency under standard conditions. If not found in data sheet, use eta_0 = ((V_mp0*I_mp0)/(1000*A_cel*n_ser))"
```

Comment:
Same as in above file, and use the right variable names.

Reply:
corrected it